validators/validator.go

```diff
@@ -9,15 +9,15 @@ import (
 
 func New() *validator.Validate {
 	validate := validator.New()
-	validate.RegisterValidation("date", isDate)
-	validate.RegisterValidation("iso3166_1_alpha2_lowercase", isIso3166Alpha2Lowercase)
-	validate.RegisterValidation("umax", uMax)
-	validate.RegisterValidation("umin", uMin)
+	_ = validate.RegisterValidation("date", isDate)
```

Comment:
I think it is: `RegisterValidation` returns an error basically on programming errors.

It should be ideally checked at compile time since we are using constants, but I think it's fair to panic at runtime if we have a bug. The error checking routine would have been an explicit exit anyway because we can't continue, something like:

```go
if (err != nil) {
    fmt.Fprint(os.Stderr, "FATAL: can't register [...]")
    os.Exit(1)
}
```

which hides the bug and isn't a whole lot different than a panic.

Reply:
I see it, in fact they are returning errors for programming errors.
Ok!